src/lib/util.ts

```diff
@@ -468,6 +468,17 @@ export function convertAttackStyleToGearSetup(style: OffenceGearStat | DefenceGe
 	return setup;
 }
 
+export function convertPvmStylesToGearSetup(attackStyles: SkillsEnum[]) {
+	const usedSetups: GearSetupType[] = [];
+	if (attackStyles.includes(SkillsEnum.Ranged)) usedSetups.push('range');
+	if (attackStyles.includes(SkillsEnum.Magic)) usedSetups.push('mage');
+	if (![SkillsEnum.Magic, SkillsEnum.Ranged].some(s => attackStyles.includes(s))) {
+		usedSetups.push('melee');
+	}
+	if (usedSetups.length === 0) usedSetups.push('melee');
+	return usedSetups;
+}
+
```

Comment:
we miiiight have another function that already does this?

Reply:
There's a similar one, but nothing that solves this problem yet.

The similar one is `convertAttackStyleToGearSetup` but 'AttackStyle' does mean the minionKill() SkillsEnum attack style, it uses GearStat source